### ISSUE TITLE
Zend OPcache Not Installed?

### ISSUE THREAD

**Issue by fdcastel** [bitnami/bitnami-docker-phabricator#42]
Phabricator `2017.02-r0` is giving me this warning:

![image](https://cloud.githubusercontent.com/assets/629408/22166152/d30cfdd2-df46-11e6-85f8-a52234cc25d7.png)

Is this extension installed in the latest bitnami images?

(I'm asking because I'm having some problems with false warning as reported in #41)

**Comment by juan131** [comment]
@fdcastel as you can check at `/opt/bitnami/php/conf/php.ini` OpCache extension is installed, configured and activated on php.
You can also check that the binary is available at `/opt/bitnami/php/lib/php/extensions/opcache.so`
On the other hand, Phabricator is configured to ignore that warning as you can see running:
```
/opt/bitnami/phabricator/bin/config get config.ignore-issues
```
When removing this "warning" from the list to ignore you will find the following error:
![screen shot 2017-01-26 at 17 01 51](https://cloud.githubusercontent.com/assets/6740773/22339098/3163cc52-e3e9-11e6-9ed9-3b274fb30b2d.png)
We have identified the problem in the php settings and we'll remove this "warning" from the "ignore list" in the new release coming and php will be properly configured.

**Comment by juan131** [comment]
Issue fixed on new release 2017.04-​r0

**Comment by juan131** [comment]
@fdcastel could you confirm us that the issue is solved on the new release?

**Comment by fdcastel** [comment]
Solved on `2017.04-​r0` 👍

**Comment by chipironcin** [comment]
Thanks @juan131 and @fdcastel 👏
